src/index.ts

```diff
@@ -0,0 +1,356 @@
+ * https://cloud.google.com/error-reporting/docs/formatting-error-messages} and
+ * the [official documentation]{@link
+ * https://cloud.google.com/error-reporting/reference/rest/v1beta1/ServiceContext}.
+ * @param {string} [options.serviceContext.service] An identifier of the
+ *     service, such as the name of the executable, job, or Google App Engine
+ *     service name.
+ * @param {string} [options.serviceContext.version] Represents the version of
+ *     the service.
+ * @param {string} [options.projectId] The project ID from the Google Cloud
+ *     Console, e.g. 'grape-spaceship-123'. We will also check the environment
+ *     variable `GCLOUD_PROJECT` for your project ID. If your app is running in
+ *     an environment which supports {@link
+ * https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application
+ * Application Default Credentials}, your project ID will be detected
+ * automatically.
+ * @param {string} [options.keyFilename] Full path to the a .json, .pem, or .p12
+ *     key downloaded from the Google Cloud Console. If you provide a path
+ *     to a JSON file, the `projectId` option above is not necessary. NOTE: .pem
+ *     and .p12 require you to specify the `email` option as well.
+ * @param {string} [options.email] Account email address. Required when using a
+ *     .pem or .p12 keyFilename.
+ * @param {object} [options.credentials] Credentials object.
+ * @param {string} [options.credentials.client_email]
+ * @param {string} [options.credentials.private_key]
+ * @param {boolean} [options.autoRetry=true] Automatically retry requests if the
+ *     response is related to rate limits or certain intermittent server errors.
+ *     We will exponentially backoff subsequent requests by default.
+ * @param {number} [options.maxRetries=3] Maximum number of automatic retries
+ *     attempted before returning the error.
+ * @param {constructor} [options.promise] Custom promise module to use instead
+ *     of native Promises.
+ *
+ * @example <caption>Import the client library</caption>
+ * const LoggingWinston = require('@google-cloud/logging-winston');
+ *
+ * @example <caption>Create a client that uses <a
+ * href="https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application">Application
+ * Default Credentials (ADC)</a>:</caption> const loggingWinston = new
+ * LoggingWinston();
+ *
+ * @example <caption>Create a client with <a
+ * href="https://cloud.google.com/docs/authentication/production#obtaining_and_providing_service_account_credentials_manually">explicit
+ * credentials</a>:</caption> const loggingWinston = new LoggingWinston({
+ *   projectId: 'your-project-id',
+ *   keyFilename: '/path/to/keyfile.json'
+ * });
+ *
+ * @example <caption>include:samples/quickstart.js</caption>
+ * region_tag:logging_winston_quickstart
+ * Full quickstart example:
+ */
+
+export class LoggingWinston extends winston.Transport {
+  inspectMetadata: boolean;
```

Comment:
Do any of these properties need to be public?

Reply:
Fixed to private.